chia/seeder/crawler.py

```diff
@@ -113,6 +123,27 @@ async def peer_action(peer: ws.WSChiaConnection):
     async def _start(self):
         self.task = asyncio.create_task(self.crawl())
 
+        # Starts the prometheus server if enabled in config
+        await self.prometheus.start_server()
+
+    async def update_metric_values(self):
+        self.prometheus.reliable_nodes.set(self.crawl_store.get_reliable_peers())
+        self.prometheus.total_5d.set(len(self.best_timestamp_per_peer))
+        ipv6_addresses_count = 0
+        for host in self.best_timestamp_per_peer.keys():
+            try:
+                _ = ipaddress.IPv6Address(host)
+                ipv6_addresses_count += 1
+            except ValueError:
```

Comment:
Unless there's a reason for `ValueError` specifically?

```suggestion
            except AddressValueError:
```

Reply:
Eh, I didn't write this part of the code initially, not sure why that was chosen

Reply:
looking at the docs, seems like it should just return the AddressValueError so I can swap it